test/Test/P/List.hs

```diff
@@ -19,6 +19,10 @@ prop_lastMaybe_empty :: Property
 prop_lastMaybe_empty =
   lastMaybe [] === (Nothing :: Maybe ())
 
+prop_count :: [Int] -> Property
+prop_count list =
+  count (>= 0) list === length (filter (>= 0) list) 
+
```

Comment:
Sorry, the inability to adequately `show` the `a -> Bool` function hurts the reproducibility of any failures, my bad, its probably not worth it @etorreborre